src/exoscale/compute/api/http.clj

```diff
@@ -105,16 +99,19 @@
 
```

Comment:
we need something like :
```
(-> (vec result)
    (with-meta (meta resp)))
```
when returning from `list-pager-fn`..but maybe there is a better place for this. obwald does not get metadata.

Reply:
nice catch @arthuraliiev, I pushed an update to address the problem.